tests/emitters/iso_line_emitter_test.py

```diff
@@ -6,12 +6,12 @@
 
 def test_properties_are_correct(archive_fixture):
```

Comment:
Change this to test `iso_sigma`, `line_sigma`, and `sigma0`. Also add a test which checks that `sigma0` equals `iso_sigma` if `sigma0` is not passed in at construction.

And add a similar test for `GaussianEmitter` which checks `sigma0` and `sigma`.